src/distributions/microsoft/installer.ts

```diff
@@ -0,0 +1,122 @@
+import { JavaBase } from '../base-installer';
+import { JavaDownloadRelease, JavaInstallerOptions, JavaInstallerResults } from '../base-models';
+import semver from 'semver';
+import { extractJdkFile, getDownloadArchiveExtension, isVersionSatisfies } from '../../util';
+import * as core from '@actions/core';
+import { MicrosoftVersion, PlatformOptions } from './models';
+import * as tc from '@actions/tool-cache';
+import fs from 'fs';
+import path from 'path';
+
+const supportedPlatform = `'linux', 'macos', 'windows'`;
+
+const supportedArchitecture = `'x64', 'armv7', 'aarch64'`;
```

Comment:
The supportedArchitecture variable is not used. Could you please remove it.

Reply:
done.